pkg/emb6/Makefile

```diff
@@ -0,0 +1,33 @@
+PKG_NAME=emb6
+PKG_URL=git://github.com/hso-esk/emb6.git
```

Comment:
In general: because it's [faster than http(s)](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#The-Git-Protocol). But when reading the doc I realized that Git made some advancements and has a Smart HTTP protocol now, too.